src/domain/createLoggerMiddleware/index.js

```diff
@@ -1,20 +1,22 @@
 import { createLogger } from 'redux-logger';
 import log from 'domain/log';
+import Config from 'domain/Config';
+import is from 'is_js';
 
 export function createLoggerMiddleware() {
-  if (PRODUCTION) {
-    return () => (next) => (action) => {
-      try {
-        log.debug('Dispatched action:', JSON.stringify(action, null, 2));
-      } catch (e) {
-        log.warn('Could not log action:', e);
-      }
-
-      return next(action);
-    };
+  if (is.not.string(Config.get('sentryDsn')) || is.empty(Config.get('sentryDsn'))) {
```

Comment:
Sure, but I was thinking it could be enabled by default with an option to turn it off